packages/cli/src/commands/generate/sdl/__tests__/fixtures/enumGeneratedSdl.js

```diff
@@ -1,4 +1,5 @@
 import gql from 'graphql-tag'
+
```

Comment:
Wait whaaaat

Reply:
This, and all the other formatting related changes, were because of `retainLines: true`